<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QWidget" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(191, 215, 234);
</string>
  </property>
  <widget class="QGraphicsView" name="image_graphicsView">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 60%;
background-color: palette(base);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_chaneImage">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 48, 73);
border-radius: 10px;
color:rgb(34, 119, 255)</string>
   </property>
   <property name="text">
    <string>Change Pic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_save">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 48, 73);
border-radius: 10px;
color:rgb(245, 223, 187);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_cancel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 48, 73);
border-radius: 10px;
color:rgb(245, 223, 187);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>331</width>
     <height>169</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="led_phonenum">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="led_username">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="led_password">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_phonenum">
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 48, 73);</string>
      </property>
      <property name="text">
       <string>Change PhoneNum :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="led_email">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="dateEdit">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(253, 240, 213);
border-radius: 10%;</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbl_email">
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 48, 73);</string>
      </property>
      <property name="text">
       <string>Change Email:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbl_password">
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 48, 73);</string>
      </property>
      <property name="text">
       <string>Change Password :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lbl_birth">
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 48, 73);</string>
      </property>
      <property name="text">
       <string>Change BirthDate:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_username">
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 48, 73);</string>
      </property>
      <property name="text">
       <string>Username :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbn_edit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 48, 73);
border-radius: 10px;
color:rgb(245, 223, 187);</string>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
